devices/tuya.js

```diff
@@ -1940,6 +1940,30 @@ module.exports = [
             await reporting.bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        fingerprint: [{modelID: 'TS0002', manufacturerName: '_TZ3000_zmy4lslw'}],
```

Comment:
This device is already supported: https://github.com/Koenkk/zigbee-herdsman-converters/blob/328ce185b1d74a57b23c2c925682aa3bb654576f/devices/tuya.js#L1975

Reply:
In my version we have also power-on behavior and switch type setting (quite important for me)

Reply:
This is also supported (when using z2m 1.29.1)